x-pack/test/apm_api_integration/tests/alerts/anomaly_alert.spec.ts

```diff
@@ -86,6 +86,7 @@ export default function ApiTest({ getService }: FtrProviderContext) {
               windowSize: 5,
               windowUnit: 'h',
               anomalySeverityType: ML_ANOMALY_SEVERITY.WARNING,
+              anomalyDetectorTypes: ['latency'],
```

Comment:
We should use the existing constants

```suggestion
              anomalyDetectorTypes: [ApmMlDetectorType.txLatency],
```

Reply:
`ApmMLDetectorType.txLatency = 'txLatency'`, that it's not what I wanted in this case

Reply:
> `ApmMLDetectorType.txLatency = 'txLatency'`, that it's not what I wanted in this case

Sorry, what do you mean?